### PR TITLE
add npm auto-update config of leaflet.fullscreen

### PR DESCRIPTION
**Pull request by maruilian11**
for #7290

### CODE REVIEW

File: ajax/libs/leaflet.fullscreen/package.json [hunk 1]
```diff
@@ -9,5 +9,14 @@
...
     "url": "https://github.com/brunob/leaflet.fullscreen"
   },
-  "author": "b_b"
+  "author": "b_b",
+  "npmName": "leaflet.fullscreen",
+  "npmFileMap": [
+    {
+      "basePath": "",
+      "files": [
+        "Control.FullScreen*.+(css|js)"
```
Comment (PeterDaveHello) [inline ajax/libs/leaflet.fullscreen/package.json]: Please add `map` with `js` and `css`.

**Comment by maruilian11** [comment]
@PeterDaveHello updated.

**Comment by PeterDaveHello** [comment]
png image missing.

**Comment by maruilian11** [comment]
@PeterDaveHello but `icon-fullscreen-2x.png` and `icon-fullscreen.png` aren't used in files we hosted.

**Comment by PeterDaveHello** [comment]
You must be kidding, see `Control.FullScreen.css`.

```css
.fullscreen-icon { background-image: url(icon-fullscreen.png); }                                                                                              
.leaflet-retina .fullscreen-icon { background-image: url(icon-fullscreen-2x.png); background-size: 26px 26px; }                                               
.leaflet-container:-webkit-full-screen { width: 100% !important; height: 100% !important; z-index: 99999; }                                                   
.leaflet-pseudo-fullscreen { position: fixed !important; width: 100% !important; height: 100% !important; top: 0px !important; left: 0px !important; z-index: 99999; }
```

**Comment by maruilian11** [comment]
@PeterDaveHello updated.

**Pull request merged**